### PR TITLE
Mt 19765 shipping discount

### PR DESCRIPTION
**Pull request by kaiyuancheng**

### CODE REVIEW

File: changelog/2024/2024-10-16-changelog.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Added Shipping discount in Promotions Builder'
+date: '2024-10-16'
+tags: ['Commerce Manager', 'Promotions Builder']
+hide_blog_post_date: false
+---
+
+**MAJOR** We have introduced a new feature allowing the creation of shipping discount for cart shipping groups. For more information on API, see [Creating a Shipping Discount](docs/promotions-builder/promotions-builder-api/promotions-builder-management/create-a-shipping-discount.mdx).
```
Comment (dazadep) [inline changelog/2024/2024-10-16-changelog.md:8]:
```suggestion
**MAJOR** We have introduced a new feature allowing the creation of shipping discount for cart shipping groups. For more information, see [Create a Shipping Discount API](docs/promotions-builder/promotions-builder-api/promotions-builder-management/create-a-shipping-discount.mdx), and [Creating a Shipping Promotion in Commerce Manager](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-a-shipping-promotion).
```

File: changelog/2024/2024-10-16-changelog.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+**MAJOR** Commerce Manager: We have added a new feature allowing you to create shipping discount in Promotions Builder. For more information, see [Creating a Shipping Promotion](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-a-shipping-promotion).
```
Comment (dazadep) [inline changelog/2024/2024-10-16-changelog.md:11]:
```suggestion
```

File: docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder.mdx [hunk 3]
```diff
@@ -149,6 +149,17 @@ To create a BXGY promotion, do the following:
...
+### Creating a Shipping Promotion
+To create a shipping discount promotion, do the following:
+
+1. Select **Shipping** in the **Actions** section.
+1. Choose **a percentage** or **an amount**
+1. Enter the value in the **value** field based on your selection in the previous step.
+1. (Optional) If there is targeted shipping type for discount, then select **+ more options**.
+1. Check the checkbox **Apply discount to**
+1. Enter the shipping type value that matches targeted shipping group and hit Enter. For example, "regular".
+1. Click **Done**.
```
Comment (wberry22) [inline docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder.mdx:161]: Very small thing, but looks like some of these end in a period and others don't.  Should we make them consistent?

File: docs/commerce-manager/promotions-builder/index.mdx [hunk 4]
```diff
@@ -19,6 +19,7 @@ Promotions in the Promotions Builder support the following actions:
     - Percentage discount: This reduces the cart total by a specific percentage, such as 10% off.
 - [Item Discount](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-an-item-discount-promotion): An item discount applies to the eligible items specified in the conditions. It can be a fixed amount, percentage, fixed price, or other discount specified such as **BXGY**.
 - [BXGY](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-a-bxgy-promotion): Buy X and Get Y (BXGY) discounts apply to the items identified within the promotional actions. This means that when a Buy X Get Y promotion is triggered, the specified items receive the associated discounts outlined in the promotion's configuration.
+- [Shipping Discount](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-a-shipping-promotion): Shipping discount applies to eligible cart shipping groups. The discount applies to shipping base price excluding tax and fees.
```
Comment (wberry22) [inline docs/commerce-manager/promotions-builder/index.mdx:22]:
```suggestion
- [Shipping Discount](/docs/commerce-manager/promotions-builder/creating-a-promotion-in-promotions-builder#creating-a-shipping-promotion): Shipping discount applies to eligible shipping groups. The discount applies to the shipping base price, excluding tax and fees.
```

File: docs/promotions-builder/promotions-builder-api/promotions-builder-api-overview.mdx [hunk 5]
```diff
@@ -4,6 +4,8 @@ nav_label: Promotions Builder API Overview
 sidebar_position: 1
 ---
 
+The Promotions Builder is a rules engine designed to help you create various types of promotions based on customizable conditions and actions. Through flexible rules and strategies, the Promotions Builder allows merchants to build complex promotions tailored to specific business needs.
```
Comment (wberry22) [inline docs/promotions-builder/promotions-builder-api/promotions-builder-api-overview.mdx:7]: I'm not sure if we want to call out the Promotions Builder as a rules engine or not (it is kind of that, but also kind of not?).

```suggestion
The Promotions Builder helps users create various types of promotions based on customizable conditions and actions. Through flexible rules and strategies, the Promotions Builder allows merchants to build complex promotions tailored to specific business needs.
```

File: docs/promotions-builder/overview.mdx [hunk 6]
```diff
@@ -4,9 +4,11 @@ nav_label: Overview
 sidebar_position: 1
 ---
 
+The Promotions Builder is a rules engine designed to help you create various types of promotions based on customizable conditions and actions. Through flexible rules and strategies, the Promotions Builder allows merchants to build complex promotions tailored to specific business needs.
```
Comment (wberry22) [inline docs/promotions-builder/overview.mdx:7]: Similar comment to one further along about whether we want to call the Promotions Builder a "rules engine".

**Review by wberry22 — APPROVED** [review]

**Pull request merged**